proxy/pkg/proxy/policy/selector_test.go

```diff
@@ -141,7 +141,7 @@ func TestClaimsSelector(t *testing.T) {
 	})
 
 	var tests = []testCase{
-		{"unatuhenticated", context.Background(), nil, "unauthenticated"},
+		{"unauthenticated", context.Background(), nil, "unauthenticated"},
```

Comment:
As this is a test, and as I've already tripped on one intentional typo testcase, it wouldn't shock me if this is wrong.

If it's wrong, it'd help if the test case clearly spelled out that it was an intentional typo.

Reply:
looks good :-)